### ISSUE TITLE
SPIKE: Assess feasibility of adding axe to cypress

### ISSUE THREAD

**Issue by georgehager** [CDCgov/prime-simplereport#4529]
We'd like to add axe accessibility testing to cypress, so that we can accurately prevent accessibility regressions on our pages. We currently are planning to use [lighthouse testing](https://github.com/CDCgov/prime-simplereport/issues/4336), however lighthouse is not as granular as we would like and will miss some issues that deque flags.

As an example, compare lighthouse to axe:
![image](https://user-images.githubusercontent.com/89797785/196788341-834ea913-9db6-4c50-bbcf-8a9b9b2182a7.png)
![image (1)](https://user-images.githubusercontent.com/89797785/196788361-4e37b936-632d-49c4-b0a2-48373fc26a94.png)

It should be possible to add basic testing to each cypress page that we test, but we need to assess the feasibility before moving ahead.

Resources:
https://timdeschryver.dev/blog/setting-up-cypress-with-axe-for-accessibility
https://www.deque.com/blog/how-to-test-for-accessibility-with-cypress/
https://www.adrianbolonio.com/blog/accessibility-github-actions

**Comment by georgehager** [comment]
I've confirmed that adding axe to cypress is very straightforward. See changes on [`george/4529-axe-spike`](https://github.com/CDCgov/prime-simplereport/tree/george/4529-axe-spike) for configuration.

Now, we just need to add `cy.injectAxe()` and `cy.checkA11y()` to each page visit in cypress.

Additionally, I've confirmed that accessibility violations _actually_ do get flagged properly:

![Screen Shot 2022-10-20 at 4 02 17 PM](https://user-images.githubusercontent.com/89797785/197047656-81c47d0d-6cf4-4f60-a7a4-a9127bfd1d42.png)

**Comment by georgehager** [comment]
@emmastephenson any other concerns before I close the spike?
